src/routes/(app)/admin[[hash=admin_hash]]/nostr/+page.server.ts

```diff
@@ -129,5 +131,34 @@ export const actions = {
 		} finally {
 			relayPool.close();
 		}
+	},
+	addRelay: async ({ request }) => {
+		const formData = await request.formData();
+		const json: JsonObject = {};
+		for (const [key, value] of formData) {
+			set(json, key, value);
+		}
```

Comment:
you don't really need JSON here, you can do `relays: formData.getAll("relays")` and in the frontend you can do `relays[]` directly in the name field (or `relays` even)